stack.sh

```diff
@@ -363,8 +363,12 @@ safe_chmod 0755 $DATA_DIR
 # Certain services such as rabbitmq require that the local hostname resolves
 # correctly.  Make sure it exists in /etc/hosts so that is always true.
 LOCAL_HOSTNAME=`hostname -s`
-if [ -z "`grep ^127.0.0.1 /etc/hosts | grep $LOCAL_HOSTNAME`" ]; then
-    sudo sed -i "s/\(^127.0.0.1.*\)/\1 $LOCAL_HOSTNAME/" /etc/hosts
+HOSTS_FILE="/etc/hosts"
+if is_clearlinux; then
+    HOSTS_FILE="/usr/share/defaults/etc/hosts"
```

Comment:
Another idea, a bit of a hack sure, is to just create a symlink in here to the actual hosts file.  Nothing else needs to change if we do that, there seems to be a lot of these exceptions for Clear...